src/features/modules.md

```diff
@@ -16,7 +16,7 @@ JavaScript modules are now [supported in all major browsers](https://caniuse.com
 <feature-support chrome="61"
                  firefox="60"
                  safari="11"
-                 nodejs="no"
+                 nodejs="13"
```

Comment:
I think this should be
```suggestion
                 nodejs="13.2"
```
or whatever version it will be released

Reply:
+1

Reply:
Done